seed/views/properties.py

```diff
@@ -623,18 +527,26 @@ def unmerge(self, request, pk=None):
                 'message': 'property view with id {} must have two parent states'.format(pk)
             }
 
-        label = apps.get_model('seed', 'PropertyView_labels')
         state1 = log.parent_state1
         state2 = log.parent_state2
         cycle_id = old_view.cycle_id
 
-        # Clone the property record, then copy over meters
+        # Clone the property record twice, then copy over meters
         old_property = old_view.property
         new_property = old_property
         new_property.id = None
         new_property.save()
 
+        new_property_2 = Property.objects.get(pk=new_property.id)
+        new_property_2.id = None
+        new_property_2.save()
+
         Property.objects.get(pk=new_property.id).copy_meters(old_view.property_id)
+        Property.objects.get(pk=new_property_2.id).copy_meters(old_view.property_id)
+
+        # If canonical Property is NOT associated to a different -View, delete it
+        if not PropertyView.objects.filter(property_id=old_view.property_id).exclude(id=old_view.id).exists():
+            Property.objects.get(pk=old_view.property_id).delete()
```

Comment:
Note: This and the same logic for Tax Lots are done to "clean up" canonical records. This technically isn't relevant until linking is introduced as canonical records only belong to one -View for now.